docs/content/preview/yugabyte-voyager/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Migrate to YugabyteDB Voyager
+headerTitle: YugabyteDB Voyager
+linkTitle: YugabyteDB Voyager
+headcontent: Simplify migration from legacy and cloud databases to YugabyteDB
+cascade:
+  unversioned: true
+description: YugabyteDB Voyager is a powerful open-source data migration engine that helps you migrate your database to YugabyteDB quickly and securely.
+type: indexpage
+menu:
+  preview_yugabyte-voyager:
+    identifier: yugabyte-voyager
+    parent: yugabytedb-voyager
+    weight: 99
+breadcrumbDisable: true
+resourcesIntro: Quick Links
+resources:
+  - title: What's new
+    url: /preview/yugabyte-voyager/release-notes/
+  - title: Download
+    url: https://download.yugabyte.com/migrate#
```

Comment:
```suggestion
    url: https://download.yugabyte.com/migrate
```

Reply:
The url appends a slash  (/) after the # making it `https://download.yugabyte.com/migrate#/`- so it points to local install instead incorrectly.